src/transformers/utils/fx.py

```diff
@@ -295,7 +374,7 @@ def path_of_module(self, mod: nn.Module) -> str:
             if path is None:
                 path = self._insert_module_as_submodule(mod)
             if path is None:
-                raise NameError("module is not installed as a submodule")
+                raise NameError("Module is not installed as a submodule")
```

Comment:
may be it'd help to print the name of the module here and another instance of the same below so that it gives context to the user? or do you think this is enough?

Reply:
You're right, I think that if that happens the current solution was to put a pdb breakpoint, printing the name of the module can help to pinpoint the origin of the problem without having to do so.

Thanks for the suggestion!